grace/evaluation/visualisation.py

```diff
@@ -0,0 +1,258 @@
+from grace.base import GraphAttrs, Annotation
+from grace.napari.utils import EdgeColor
+from collections import Counter
+
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy as np
+
+import numpy.typing as npt
+
+from skimage.util import montage
+
+
+def plot_simple_graph(G: nx.Graph, title: str = "") -> None:
+    """Plots a simple graph with black nodes and edges."""
+
+    # Fancy annotation plot
+    _, ax = plt.subplots(figsize=(16, 16))
+
+    # node positions
+    pos = {
+        idx: (node[GraphAttrs.NODE_X], node[GraphAttrs.NODE_Y])
```

Comment:
Graphs have the X and the Y swapped, so when plotting this, the graph is transposed to the original image

Reply:
This will get fixed in the subsequent PR.